src/chainparamsbase.cpp

```diff
@@ -91,12 +91,19 @@ std::string ChainNameFromCommandLine()
 {
     bool fRegTest = gArgs.GetBoolArg("-regtest", false);
     bool fTestNet = gArgs.GetBoolArg("-testnet", false);
+    bool fMainNet = gArgs.GetBoolArg("-mainnet", false);
+
+#ifdef ENABLE_MAINNET
+    fMainNet = true;
+#endif
 
     if (fTestNet && fRegTest)
         throw std::runtime_error("Invalid combination of -regtest and -testnet.");
     if (fRegTest)
         return CBaseChainParams::REGTEST;
     if (fTestNet)
         return CBaseChainParams::TESTNET;
-    return CBaseChainParams::MAIN;
+    if (fMainNet)
+        return CBaseChainParams::MAIN;
+    return CBaseChainParams::TESTNET;
```

Comment:
Is silently running testnet really the best solution? This is going to confuse the heck out of me :)

Reply:
I agree, will be an adjustment for anyone accustomed to running on `mainnet` when building from source.